lib/run/index.js

```diff
@@ -302,7 +304,10 @@ module.exports = function (options, callback) {
                     // in case runtime faced an error during run, we do not process any other event and emit `done`.
                     // we do it this way since, an error in `done` callback would have anyway skipped any intermediate
                     // events or callbacks
-                    if (err) {
+
+                    // TODO: Add analytics call here
+
+                    if (err) { // TODO -> understand when this is raised
```

Comment:
This error is raised from the `postman-runtime` library if the collection run did not finish successfully.